CMakeLists.txt

```diff
@@ -155,6 +155,7 @@ include(TestBigEndian)
 test_big_endian(WORDS_BIGENDIAN)
```

Comment:
-	pd = pcap_open_live(props.path.c_str(), SnapLen(), 1, 1, tmp_errbuf);
rsmmr added a note
7 days ago
Can you elaborate why you replaced the pcap_open_live call with the individual calls below? Does this raise the version of libpcap required?
knielander added a note
4 days ago
It doesn't raise the version, but it will allow you to specify the ring buffer size.